templates/homepage.hbs

```diff
@@ -1,5 +1,20 @@
 {{#extend "page"}}
 {{#content "main"}}
+  {{!-- <div class="container demo-1">
```

Comment:
I think this commented out block can be removed now. The newer hero markup + Gravity CSS are working now.